src/panels/config/automation/action/ha-automation-action.ts

```diff
@@ -279,6 +279,11 @@ export default class HaAutomationAction extends LitElement {
           margin-bottom: 16px;
           scroll-margin-top: 48px;
         }
+        ha-svg-icon[slot="icon"] {
```

Comment:
Why this one? It doesn't have this in ltr now? (also, make the selector more specific)